src/hidimstat/utils.py

```diff
@@ -69,8 +142,60 @@ def cal_fdp_power(selected, non_zero_index, r_index=False):
     return fdp, power
 
 
+def fdr_threshold(pvals, fdr=0.1, method="bhq", reshaping_function=None):
+    """
+    False Discovery Rate thresholding method
+
+    Parameters
+    ----------
+    pvals : 1D ndarray
+        set of p-values
+    fdr : float, default=0.1
+        False Discovery Rate
+    method : str, default='bhq'
+        Method to control FDR. Available methods are 'bhq', 'bhy', 'ebh'
```

Comment:
Maybe we could explicit the methods acronyms in the docstring.

Reply:
done